addons/pause/userscript.js

```diff
@@ -15,6 +15,14 @@ export default async function ({ addon, console, msg }) {
   setSrc();
   onPauseChanged(setSrc);
 
+  if (addon.settings.get("auto-pause")) document.addEventListener("focusout", () => setPaused(true));
+  addon.settings.addEventListener("change", () => {
+    console.log("Settings changed!");
+    if (addon.settings.get("auto-pause") === true) document.addEventListener("focusout", () => setPaused(true));
+    else if (addon.settings.get("auto-pause") === false)
+      document.removeEventListener("focusout", () => setPaused(false));
```

Comment:
I don't think putting an else on the next line like that works. You'll probably have to add the braces to the condition or use a [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator).

Reply:
> I don't think putting the else on the next line without braces works. 

I don't see why it wouldn't work.